client/src/App.js

```diff
@@ -6,12 +6,15 @@ import Channels from "./pages/Channels";
 import Cohorts from "./pages/Cohorts";
 import Channel from "../src/pages/Channel";
 import Login from "./pages/Login";
-import Logout from "./pages/Logout";
-
+import Logout from "./components/Logout";
+import SignUp from "./pages/SignUp";
 const App = () => {
 	const [token, setToken] = useState(false);
```

Comment:
why are you passing the `setToken` variable to every single component, it is only being used by Logout, Login and SignUp